src/navigators/LoggedInStack.tsx

```diff
@@ -27,6 +28,7 @@ export function LoggedInStack() {
   } = useActiveAccount();
   const {
     activeProject,
+    activeSubjectId,
```

Comment:
Last one- I don't think we're using this in a meaningful way and this can be removed, right?